### ISSUE TITLE
referencenet initializing warning ?

### ISSUE THREAD

**Issue by lbwang2006** [guoqincode/AnimateAnyone-unofficial#51]
Some weights of the model checkpoint were not used when initializing ReferenceNet: 
 ['conv_norm_out.bias, conv_norm_out.weight, conv_out.bias, conv_out.weight, up_blocks.3.attentions.2.proj_out.bias, up_blocks.3.attentions.2.proj_out.weight, up_blocks.3.attentions.2.transformer_blocks.0.attn1.to_k.weight, up_blocks.3.attentions.2.transformer_blocks.0.attn1.to_out.0.bias, up_blocks.3.attentions.2.transformer_blocks.0.attn1.to_out.0.weight, up_blocks.3.attentions.2.transformer_blocks.0.attn1.to_q.weight, up_blocks.3.attentions.2.transformer_blocks.0.attn1.to_v.weight, up_blocks.3.attentions.2.transformer_blocks.0.attn2.to_k.weight, up_blocks.3.attentions.2.transformer_blocks.0.attn2.to_out.0.bias, up_blocks.3.attentions.2.transformer_blocks.0.attn2.to_out.0.weight, up_blocks.3.attentions.2.transformer_blocks.0.attn2.to_q.weight, up_blocks.3.attentions.2.transformer_blocks.0.attn2.to_v.weight, up_blocks.3.attentions.2.transformer_blocks.0.ff.net.0.proj.bias, up_blocks.3.attentions.2.transformer_blocks.0.ff.net.0.proj.weight, up_blocks.3.attentions.2.transformer_blocks.0.ff.net.2.bias, up_blocks.3.attentions.2.transformer_blocks.0.ff.net.2.weight, up_blocks.3.attentions.2.transformer_blocks.0.norm2.bias, up_blocks.3.attentions.2.transformer_blocks.0.norm2.weight, up_blocks.3.attentions.2.transformer_blocks.0.norm3.bias, up_blocks.3.attentions.2.transformer_blocks.0.norm3.weight']

is this correct？the training loss is not decreasing，result：
![grid](https://github.com/guoqincode/AnimateAnyone-unofficial/assets/7641281/7002a456-a713-4d9c-a01e-173490ebcffc)

the pose condition is invalid..

**Comment by guoqincode** [comment]
1. This is correct.
2. Please train more steps in high resolution.
3. If you find our project helpful, please click a star, thank you.

**Comment by lbwang2006** [comment]
> 1. This is correct.
> 2. Please train more steps in high resolution.
> 3. If you find our project helpful, please click a star, thank you.

have starred! great work

I trained with 768x512, 64 batchsize, 10000 step result, is this ok? should I continue to train or the 10000 step already bad and tune some setting?

**Comment by lbwang2006** [comment]
the pose condition looks invalid,

**Comment by guoqincode** [comment]
During my training, I observed that the model learned the reference features before the pose features, and the pose features were learned later.

**Comment by lbwang2006** [comment]
finally the first stage training failed , the pose condition looks invalid, the max output of poseguilder is much smaller than latent input, I think the learning rate of poseguider needs a bit higher?
